pandas/io/parsers.py

```diff
@@ -406,6 +438,10 @@ def _read(filepath_or_buffer, kwds):
     chunksize = _validate_integer('chunksize', kwds.get('chunksize', None), 1)
     nrows = _validate_integer('nrows', kwds.get('nrows', None))
 
+    # Check for duplicates in names.
+    names = kwds.get("names", None)
+    _check_dup_names(names)
```

Comment:
call this ``_validate_names`` and have it return names, so its a similar patter to the other validators

Reply:
Done.